plaso/storage/redis/reader.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""Redis reader."""
+from __future__ import unicode_literals
+
+from plaso.lib import definitions
+from plaso.storage.redis import redis_store
+from plaso.storage import interface
+
+class RedisStorageReader(interface.StorageReader):
+  """Redis storage file reader."""
+
+  def __init__(self, task):
+    super(RedisStorageReader, self).__init__()
+    self._store = redis_store.RedisStore(
+        storage_type=definitions.STORAGE_TYPE_TASK,
+        session_identifier=task.session_identifier,
+        task_identifier=task.identifier)
+
+  def IsFinalized(self):
```

Comment:
Should a lot of these proxy methods be moved to a shared StorageReader interface?

Reply:
No, because this still needs to be called by the writer, or other clients.

Reply:
ack